back/lib/participation_method/voting.rb

```diff
@@ -20,6 +20,15 @@ def assign_defaults_for_participation_context
       end
     end
 
+    def budget_in_form?(user)
+      if participation_context.project.continuous? \
```

Comment:
Is the \ required here? Not sure what preferred style is but I would usually put && on the previous line instead

Reply:
Yeah, personal preference really. Happy to leave as it is